### ISSUE TITLE
hide object property based on schema option

### ISSUE THREAD

**Issue by tafelito** [josdejong/jsoneditor#1008]
Is there a way to hide a field from the json bases on some options? 

I want to have a field property that sets the order of the field automatically when dragging but to be hidden from the user, so I can always maintain the same order

**Comment by josdejong** [comment]
Maybe you can use the `onClassName` option ([see docs](https://github.com/josdejong/jsoneditor/blob/develop/docs/api.md)) to give this field a specific CSS class, and you can define some CSS to hide this field? I'm not sure if that will work out nicely, it may give weird issues when dragging over this field.

**Comment by JoshuaDeanEssium** [comment]
This doesnt solve the issue because when you edit the parent the class is removed and "onClassName" is not called again right away. This would allow the user to expand the child that is supposed to be hidden because the class reverts to normal and the caret shows back up. Why not just use the schema option of hidden? it feels like if this is a schema based json editor it should use these options, not just for validation. 

Has anything changed to allow this functionality? Trying to work around this is tedious and buggy

**Comment by tafelito** [comment]
yes that was not a viable solution for us either, we ended up not relying on this

**Comment by josdejong** [comment]
> Why not just use the schema option of hidden?

I wasn't aware JSON Schema has a `hidden` option. I cannot directly find more information about it, but that sounds like a great solution. 

Anyone interested in looking into that?
